config/jobs/kubernetes-csi/csi-driver-nfs/csi-driver-nfs-config.yaml

```diff
@@ -29,3 +29,27 @@ presubmits:
         resources:
           requests:
             cpu: 2000m
+  - name: pull-csi-driver-nfs-sanity
```

Comment:
This file is auto generated by gen-jobs.sh. @pohly do you have any suggestions on how to add custom tests?

Reply:
and what about e2e tests support in the future, some driver supports, some driver does not support that. Cannot be all generated by gen-jobs.sh right?

Reply:
I figured it out, just move this config to another new file.